src/pages/MissionAndPurpose/MissionAndPurpose.scss

```diff
@@ -0,0 +1,37 @@
+.about-mission-purpose{
+  font-weight: 400; 
+  text-align: left;
```

Comment:
Same with this text-align

Reply:
I needed the left or else it's center aligned